slides/markdown/kick_off.md

```diff
@@ -26,9 +26,15 @@ DO500 <!-- {_class="title-color"} -->
 
 
 <!-- .slide: data-background-image="images/chef-background.png" class="white-style" -->
-### DevOps practices used in this section:
+### DevOps practices introduced in this section:
```

Comment:
@rmarting I am not a native speaker...so not sure what is best, "used" or "introduced".  I would rather keep the "used", at least for consistency with the rest of the content.

Reply:
Thanks @jtudelag ! It makes sense your suggestion. Applied